packages/react-storage/src/components/StorageBrowser/views/LocationActionView/DeleteView/DeleteView.tsx

```diff
@@ -53,19 +53,34 @@ export const DeleteView = ({
         onActionStart={onActionStart}
         onActionCancel={onActionCancel}
       >
-        <Exit onClick={onExit} disabled={isProcessing} />
+        <ViewElement className={`${AMPLIFY_CLASS_BASE}__navigation`}>
+          <Exit onClick={onExit} disabled={isProcessing} />
+        </ViewElement>
         <Title />
-        <ViewElement className={`${CLASS_BASE}__table-wrapper`}>
+
+        <ViewElement className={`${AMPLIFY_CLASS_BASE}__table__wrapper`}>
           <DataTableControl
             className={`${CLASS_BASE}__delete-view-data-table`}
           />
         </ViewElement>
-        <ViewElement className={`${CLASS_BASE}__action-footer`}>
+
+        <ViewElement className={`${AMPLIFY_CLASS_BASE}__action__summary`}>
```

Comment:
How do we determine when a classname should have the `action` component name?

Reply:
If its only used in an action view. The title and navigation are used in all views so those are just `${AMPLIFY_CLASS_BASE}__title`. Action views have these areas that are specific to action views: options (like overwrite, add files), summary, status, message, footer, buttons. I guess if we really wanted to we could just remove any view names

Reply:
Ideally we want a pattern that is easy to keep consistent and extend in the future. If the determining factor of when a classname includes `action` is whether the corresponding component's contents are currently scoped to a single use case, it will be simpler to maintain consistency by dropping the `action` portion altogether.

Tangential: Currently all the completed composables include a wrapping `div` for positioning, do we need this additional wrapper at all?